lib/ts/recipe/emailpassword/utils.ts

```diff
@@ -60,12 +60,7 @@ export function validateAndNormaliseUserInput(
          * createAndSendCustomEmail implementation which calls our supertokens API
          */
         if (emailService === undefined) {
-            emailService = new BackwardCompatibilityService(
```

Comment:
what about changing the types of BackwardCompatibilityService?

Reply:
Done